lib/multicell_location/multicell_location.c

```diff
@@ -180,9 +231,11 @@ int multicell_location_get(const struct lte_lc_cells_info *cell_data,
 	LOG_DBG("Generated request:\n%s", log_strdup(http_request));
 
 	err = execute_http_request(http_request, strlen(http_request));
-	if (err) {
+	if (err && (err != -ETIMEDOUT)) {
 		LOG_ERR("HTTP request failed, error: %d", err);
 		return err;
+	} else if (err == -ETIMEDOUT) {
+		LOG_WRN("Data reception timed out, attempting to parse possibly incomplete data");
```

Comment:
This could be simplified a bit by switching the order of the conditions.
```suggestion
	if (err == -ETIMEDOUT) {
		LOG_WRN("Data reception timed out, attempting to parse possibly incomplete data");
	} else if (err) {
		LOG_ERR("HTTP request failed, error: %d", err);
		return err;
```